<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pb_open">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>20</y>
      <width>141</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Kalenderdatei einlesen</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_new_event">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>20</y>
      <width>111</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>termin erstellen</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_fertig">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>20</y>
      <width>80</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>fertig</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>70</y>
      <width>741</width>
      <height>441</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>Starttag</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Startzeit</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Endtag</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>EndZeit</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Ort</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Titel</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Beschreibung</string>
     </property>
    </column>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
